products/ga_s2_gm.odc-product.yaml

```diff
@@ -75,3 +75,28 @@ measurements:
   units: '1'
   dtype: uint16
   nodata: 0
+
+- name: 'SMAD'
+  aliases: [smad, sdev, SDEV]
+  units: '1'
+  dtype: uint16
```

Comment:
All *MAD bands are `float32` and do not have `nodata` as they use `NaN` to encode missing data.

There is still some confusion as to what `nodata` should be when describing floating point data.

Options are:

```yaml
    nodata: null
    nodata: .nan
```

This impacts data loading and fusing, errors in configuration are not obvious or reported until weirdness is observed during data loading, and only the regions of data overlap/missing data. Not sure what products we can use as reference for how floating point data is described.

Reply:
`ga_ls8c_wofs_2_annual_summary.yaml` product uses `nodata: .na`

Reply:
`ga_ls8c_ard_3` also uses `nodata: .nan`